libs/core/src/lib/tabs/tab-list.component.ts

```diff
@@ -464,4 +463,17 @@ export class TabListComponent implements AfterContentInit, AfterViewInit, OnDest
         && expand !== tabPanel.expanded
         && expand === false ? this.collapsibleTabs : true;
     }
+
+    /** @hidden */
+    private scrollTo(containerElement: Element, distanceToScroll: number): void {
```

Comment:
should be prefixed with `_` since it's private

Reply:
Not applicable anymore, method moved to utils